docs/features/kubernetes/configuration.md

```diff
@@ -219,6 +219,12 @@ The custom resource's apiVersion.
 
 The plural representing the custom resource.
 
+### `apiVersionOverrides` (optional)
+
+Overrides for the API versions used to make requests for the corresponding
+objects. If using a legacy Kubernetes version, you may use this config to
+override the default API versions to ones that are supported by your cluster.
```

Comment:
I think it would be good to have one or two examples of valid `apiVersion` values here

Reply:
Sure! Will add!